sass/includes/_card-groups.scss

```diff
@@ -28,13 +29,26 @@
     }
 
     &__title-link {
-      @extend .tna-card__link;
+      font-family: $font__body;
+      font-weight: $font-weight-bold;
+      font-size: 1.125rem;
```

Comment:
could use the font mixin here

Reply:
Agreed to leave this for now until the font sizes have been updated to reflect latest designs 👍